src/lib/components/progressBar/__snapshots__/progressBar.test.js.snap

```diff
@@ -0,0 +1,112 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
We should put the files **/__sanpshots__/** in the gitignore, right?

Reply:
no, snapshots must be commited